modules/mobile/app/serializers/mobile/v0/attachment_serializer.rb

```diff
@@ -8,7 +8,7 @@ class AttachmentSerializer < ActiveModel::Serializer
       attribute :id
       attribute :name
       attribute :message_id
-      attribute :attachment_size
+      attribute :attachment_size, if: -> { object.attachment_size.positive? }
```

Comment:
Looks good but we should add some unit tests.  Also, is there any scenario where `attachment_size` could be null/nil?

Reply:
If it was nil, positive would still test false and we would still not be included. Is that not what we want to do?

Reply:
I think if the value is `nil` and you try to run `.positive?` on it, it'll throw `NoMethodError (undefined method 'positive?' for nil:NilClass)`

Reply:
I think if you do this it'll handle a `nil` scenario gracefully: `object.attachment_size&.positive?`.

Source: https://stackoverflow.com/questions/12425465/ruby-check-if-nil-before-calling-method